src/components/HeaderBar/HeaderBar.tsx

```diff
@@ -1,19 +1,24 @@
-import { Navbar, Container, ButtonGroup, Button } from 'react-bootstrap';
+import { Navbar, Container, ButtonGroup, Button, useAccordionButton } from 'react-bootstrap';
 import { Link, useHistory } from "react-router-dom";
 import { useWalletHook } from '../NavBar/useWallethook';
+import { Hamburger } from '../HumburgerMenu/HumburgerMenu';
+import { useState } from 'react';
 
 export const HeaderBar: React.FC = () => {
     const history = useHistory();
     const wallet = useWalletHook();
+    const HumburgerMenu = useState();
 
     return (
         <Navbar bg="dark" variant="dark">
             <Container>
+
                 <Link to={'/'} style={{ textDecoration: 'none', color: 'black' }}>
                     <Navbar.Brand>Intern X Marketplace</Navbar.Brand>
                 </Link>
                 {wallet?.walletInfo?.address &&
                     <ButtonGroup>
+                        <Hamburger open={true} />
```

Comment:
Then you need to use those states from above like this:
```suggestion
                        <Hamburger open={isOpen}  setIsOpen={!isOpen} />
```

Reply:
<img width="1792" alt="Screen Shot 2022-09-08 at 11 50 10 am" src="https://user-images.githubusercontent.com/88649042/189016283-ccbc0ed8-0939-4874-8643-baf2f1aaad0f.png">
It gives error.

Reply:
so, I did not change it